### PR TITLE
refactor(workspace): refactor list item action

### PR DESCRIPTION
**Pull request by petrjasek**
it is a single dropdown instead of 3 buttons + dropdown

SD-2715

### CODE REVIEW

File: client/app/scripts/superdesk-archive/module.js [hunk 1]
```diff
@@ -275,10 +275,11 @@ define([
                                 });
                         }]
                 })
-                .activity('Re-write', {
+                .activity('rewrite', {
                     label: gettext('Re-write'),
                     icon: 'multi-star-color',
                     filters: [{action: 'list', type: 'archive'}],
+                    group: 'corrections',
```
Comment (mdhaman) [inline client/app/scripts/superdesk-archive/module.js]: Is the `group: 'corrections'` for grouping post publish action such as correction, kill, re-write and re-open (which is new take on the published item)?  If yes, then we would need `group: 'corrections'` for other actions as well.

Comment (petrjasek) [inline client/app/scripts/superdesk-archive/module.js]: right it is. will update

**Comment by akintolga** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**